modules/update-upgrading-cli.adoc

```diff
@@ -56,7 +56,7 @@ $ oc get clusterversion -o json|jq ".items[0].spec"
 +
```

Comment:
Also a few lines up, `confirm that your channel is set to `stable-4.5`:`, which probably also needs to be softened.

And do we want to overhaul this to use `{product-version}`?

Reply:
@wking do you mean we should not require `stable-*` only, but also give the option for `fast-*`?
>confirm that your channel is set to either `stable-4.5` or `fast-4.5`:

I guess this kind of refers back to the original question. Is it OK to also recommend fast channels during upgrade via CLI?

Reply:
Yup, releases and recommended updates in fast-* and stable-* channels are all explicitly supported, as described in the channel docs in this repo.